operations/helm/charts/mimir-distributed/templates/ingester/ingester-statefulset.yaml

```diff
@@ -57,8 +57,20 @@ spec:
     metadata:
       labels:
         {{- include "mimir.podLabels" $args | nindent 8 }}
+        {{- if (eq $rolloutZone.noDownscale true )}}
+        grafana.com/no-downscale: {{$rolloutZone.noDownscale}}
+        {{- else }}
+        grafana.com/prepare-downscale: {{$rolloutZone.prepareDownscale}}
+        {{- end }}
       annotations:
         {{- include "mimir.podAnnotations" $args | nindent 8 }}
+        {{- if $rolloutZone.prepareDownscale }}
+        grafana.com/prepare-downscale-http-path: ingester/prepare-shutdown
+        grafana.com/prepare-downscale-http-port: {{ include "mimir.serverHttpListenPort" . }}
+        {{- end-}}
+        {{- if $rolloutZone.downscaleLeader }}
+        grafana.com/rollout-downscale-leader: $rolloutZone.downscaleLeader
+        {{- end }}
```

Comment:
we should also set the `grafana.com/min-time-between-zones-downscale=12h` label on ingesters. store-gateways can work with about 30m (in case lazy loading is disabled).  alertmanagers shouldn't need this config